src/utilities/texture_atlas.jl

```diff
@@ -512,7 +511,7 @@ end
 
 function offset_bezierpath(atlas::TextureAtlas, bp::BezierPath, markersize::Vec2, markeroffset::Vec2)
     bb = bbox(bp)
-    pad_offset = (origin(bb) .- 0.5f0 .* bezierpath_pad_scale_factor(atlas, bp) .* widths(bb))
+    pad_offset = origin(bb) .- 0.5f0 .* bezierpath_pad_scale_factor(atlas, bp) * widths(bb)
```

Comment:
Yeah it's not necessary... But lets add it back, I think it relies on half deprecated functionality :D 
Would need to look at it in detail, to see what it dispatches to, but I think it's also clearer since it's certainly an element wise multiply!